compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -85,16 +88,24 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
       reorderStr = "transpose";
     } else if (options.reorderStrategy == ReorderWorkgroupsStrategy::Swizzle) {
       reorderStr = "swizzle";
+    } else if (options.reorderStrategy ==
+               ReorderWorkgroupsStrategy::ChipletGroup) {
+      reorderStr = "chilpletgroup";
     } else {
       assert(options.reorderStrategy == ReorderWorkgroupsStrategy::None &&
              "Unhandled reorder option");
       reorderStr = "none";
     }
   }
 
+  StringRef tilesizeStr = "<not set>";
+  if (options.reorderWgLogTileSize)
+    tilesizeStr = StringRef(
+        std::to_string((unsigned long)options.reorderWgLogTileSize.value()));
```

Comment:
This won't work, `tilesizeStr` will form a dangling reference because `to_string` returns a temporary